containers/attributes/main.py

```diff
@@ -812,6 +808,7 @@ async def create_attributes_definitions_crud(request, decoded_token=None):
         }
     },
 )
+@hook_into(HttpMethod.PUT, run_pre_hooks, run_post_hooks, run_err_hooks)
```

Comment:
If you take the other suggestions this will be something like:

```suggestion
@hook_into(HttpMethod.PUT, post=audit_hook, err=audit_failure_hook)
```